Examples/Retired iOS Demo/Retired iOS Demo/AppDelegate.swift

```diff
@@ -50,4 +53,8 @@ extension Message {
   private func goToAppStore(action: UIAlertAction) {
     UIApplication.sharedApplication().openURL(iTunesURL)
   }
+
+  private func doItAgainLater(action: UIAlertAction) {
+    Retired.suppressUntil(NSDate(timeIntervalSinceNow: intervalBetweenRequests))
+  }
```

Comment:
Yeah, making the "retry" thing configurable. So it handles it itself whenever a user presses cancel